src/EditorFeatures/CSharpTest/ImplementAbstractClass/ImplementAbstractClassTests.cs

```diff
@@ -1818,6 +1818,45 @@ public override void M<T>()
     {
         throw new System.NotImplementedException();
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementAbstractClass)]
+        public async Task TestFlowAnylasisAttributeOnValueType()
```

Comment:
Overall, this test seems like a good test to have.

❔ Do we have the same test for `where T : class`?
❔ Do we have the same test for `where T : class?`?

Reply:
Added tests for `string` and `string?`.

Reply:
Added.